br/pkg/logutil/logging.go

```diff
@@ -380,3 +380,14 @@ func MarshalHistogram(m prometheus.Histogram) zapcore.ObjectMarshaler {
 		return nil
 	})
 }
+
+// OverrideLevel temporary sets level to the global logger.
+//
+// Usage: `defer OverrideLevel(zapcore.LevelXxx)()`
+//
+// Don't use this in parallel, or the log level may not be reset correctly.
+func OverrideLevel(lvl zapcore.Level) func() {
```

Comment:
Can also add `testing.T` to arguments and use `t.Cleanup`, if the caller function is a child function of testing so defer will be executed when child function returns.